docs/HyBIG-Example-Usage.ipynb

```diff
@@ -57,7 +57,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from harmony import Collection, Environment, Client, Request\n",
+    "from harmony_service_lib import Collection, Environment, Client, Request\n",
```

Comment:
So happy we can do this now. Do we need a ticket to update this for all of our regression notebooks, or just sneak it in as we go along?

Reply:
You could write a ticket, but I'm just dropping it in where I can because those tickets will never get prioritized and if you want to wait until IP sprint I suppose you could.

Reply:
The elusive IP sprint. I'll just update when it comes up. If I update the granule used in the Trajectory Subsetter regression test, then that'll be a good time.

Reply:
Just spotting this thread as I opened up the PR - don't the regression test notebooks use the `harmony-py` library, rather than the `harmony-service-lib-py`? (If so, that confirms just how much of a good change this renaming is!)

Reply:
Well I wouldn't know, but that makes sense they would use harmony-py.  This has confused me for the entire time I've been on this project.

Reply:
Hm, looking at the harmony-py and harmony-service-lib-py repositories, how did the requests pass if Collection, Environment, Client, Request are all in harmony-py?